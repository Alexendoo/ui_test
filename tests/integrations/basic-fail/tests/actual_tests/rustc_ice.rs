//@compile-flags: -Ztreat-err-as-bug
//@rustc-env: RUSTC_BOOTSTRAP=1
//@normalize-stderr-test: " +[0-9]+: .*\n" -> ""
//@normalize-stderr-test: "                +at /.*\n" -> ""
//@normalize-stderr-test: " running on .*" -> ""
use basic_fail::add;

fn main() {
    add("42", 3);
    //~^ ERROR: mismatched types
}
