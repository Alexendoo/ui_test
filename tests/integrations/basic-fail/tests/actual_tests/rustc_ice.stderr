error: internal compiler error: no errors reported for args
 --> $DIR/rustc_ice.rs:8:5
  |
8 |     add("42", 3);
  |     ^^^^^^^^^^^^

thread 'rustc' panicked
